### PR TITLE
Reduce ocdb Altium symbol size by 254

### PR DESCRIPTION
**Pull request by PhilippeFerreiraDeSousa**
# Before

# After

image

# In Altium

The layout is different, run on Erwin's machine

image

### CODE REVIEW

File: utils/symbols.stanza [hunk 1]
```diff
@@ -63,76 +63,76 @@ public pcb-symbol altium-power-arrow-sym :
...
 public pcb-symbol altium-power-gnd-signal-sym :
   name = "ALTIUM-POWER-GND-SIGNAL"
   pin p[0] at unit-point(0.0, 0.0)
 
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-triangle([1.27, 0.0], [1.5 * 2.794, 0.0], 1.5 * 1.524)
-  unit-val([2.5, -1.0])
+  unit-line([[0.0, 0.0], [0.2, 0.0]])
+  unit-triangle([0.2, 0.0], [1.5 * 1.1, 0.0], 1.5 * 0.6)
+  unit-val([0.4, -0.16])
```
Comment (PhilippeFerreiraDeSousa) [inline utils/symbols.stanza:127]: Those 3 lines are wrong, I went over and divided by 2.54 twice for most values

File: utils/symbols.stanza [hunk 2]
```diff
@@ -63,76 +63,76 @@ public pcb-symbol altium-power-arrow-sym :
...
 
 public pcb-symbol altium-power-gnd-earth-sym :
   name = "ALTIUM-POWER-GND-EARTH"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val u = 0.5 * 1.27
+  val u = 0.5 / 2.0
   val ex = 4.0 * u
   unit-line([[0.0, 0.0], [ex, 0.0]])
   unit-line([[ex, u], [ex, -1.0 * u]])
```
Comment (PhilippeFerreiraDeSousa) [inline utils/symbols.stanza:138]: Just below line 143 it divides to 0.2 in unit-val

File: utils/symbols.stanza [hunk 3]
```diff
@@ -63,76 +63,76 @@ public pcb-symbol altium-power-arrow-sym :
...
 public pcb-symbol altium-power-bar-sym : 
   name = "ALTIUM-POWER-BAR"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val ex = 3.0 / 2.0 * 1.27
+  val ex = 3.0 / 2.0 * 0.5
   unit-line([[0.0, 0.0], [ex, 0.0]])
-  unit-line([[ex, -1.27 / 2.0], [ex, 1.27 / 2.0]])
-  unit-val([0.5, 0.5])
+  unit-line([[ex, 0.5 / 2.0], [ex, 0.5 / 2.0]])
```
Comment (PhilippeFerreiraDeSousa) [inline utils/symbols.stanza:88]: First one should be -0.5 and not 0.5
This is why in the visualiser the line is the wrong direction

**Review by PhilippeFerreiraDeSousa — COMMENTED** [review]
Commented errors

File: utils/symbols.stanza (hunk 2 above)
Comment (erwin-lau) [reply utils/symbols.stanza:138]: changed and used `altium-power-gnd-earth-sym` and see this. It is good.
image

**Review by erwin-lau — APPROVED** [review]
Updated according to conversations

File: utils/symbols.stanza [hunk 4]
```diff
@@ -63,135 +63,135 @@ public pcb-symbol altium-power-arrow-sym :
...
 
 public pcb-symbol altium-power-gnd-signal-sym :
   name = "ALTIUM-POWER-GND-SIGNAL"
   pin p[0] at unit-point(0.0, 0.0)
 
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-triangle([1.27, 0.0], [1.5 * 2.794, 0.0], 1.5 * 1.524)
-  unit-val([2.5, -1.0])
+  unit-line([[0.0, 0.0], [0.51, 0.0]])
+  unit-triangle([0.51, 0.0], [1.5 * 2.8, 0.0], 1.5 * 1.5)
```
Comment (PhilippeFerreiraDeSousa) [inline utils/symbols.stanza:126]: Thx just those 2.8 is 1.1 and 1.5 is 0.6

**Pull request merged**